### DISCUSSION TITLE
Convention sur le format des fichiers

### DISCUSSION THREAD

**Discussion by blanchette31** [BIO6032/2022-SEIR-reproduction#3] [General]
- Quel format allons nous utiliser pour les fichiers? Par exemple, le diagramme/ figure 1

- pdf, jpeg, png...?

Et pour les fichiers de données, par exemple, .csv?

- ...

**Comment by Marine1719** [comment]
Bonjour, 
Je pense qu'on devrait plutôt partir sur png ou jpeg, l'un ou l'autre me semble pareil. Est-ce que tout le monde est ok avec ça ? Si on choisit PNG par exemple, liker peut-être pour dire oui ou commenter si non.

**Reply by blanchette31** [reply]
PNG me convient également.

**Reply by aushay** [reply]
TIFF, EPS, PDF, et PPT sont les formats suggérés par PNAS je pense

**Reply by blanchette31** [reply]
On doit trouver les recommandations de ReScience C étant donné que c'est le journal qu'on va faire la soumission de publication. Il se peut qu'ils recommandent les mêmes formats, mais il se peut que non aussi, les journaux ont souvent des critères de soumissions différents.

**Reply by blanchette31** [reply]
J'imagine que ça doit être à quelque part dans le repository de submissions je vais refaire le tour pour essayer de trouver.

**Reply by Marine1719** [reply]
Brandon, oui je pense aussi mais j'avoue ne pas trouver, je ne sais pas si on cherche au bon endroit étant donné qu'il y a un paquet d'infos ! Dis nous si tu trouves qql chose je vais rechecker également !

**Comment by Marine1719** [comment]
Est-ce que qql a trouvé les recommandations pour les formats fichiers/figures? J'ai cherché sur le site et la page github de rescience, il y a un tas d'informations avec des codes etc pour lesquels honnêtement je comprends pas tout.. Mais je ne trouve pas les recommandations, qql peut il aider ???

**Reply by blanchette31** [reply]
Moi aussi j'ai regardé dans les différents repositories de ReScience aussi et je ne les trouvent pas. Ça ne semble pas être sur le site web du journal non plus.

**Reply by Campeon-del-Mundo** [reply]
Il n'y a pas de préférence je crois. C'est aléatoire de ce que j'ai vu sur les autres GitHub.
PNG et PDF: https://github.com/shin-ee-chen/Generative-causal-explanations-of-black-box-classifiers/tree/main/figures/FMNIST_034/pretrained
CSV et TSV: https://github.com/elfrink1/FACT/tree/main/Data/heart
